tests/test_taar_legacy.py

```diff
@@ -0,0 +1,84 @@
+"""Test suite for TAAR Legacy Job."""
+
+import pytest
+import requests
+from mozetl.taar import taar_legacy
+
+FAKE_LEGACY_DATA = {
+    "guid-01":
+        ["guid-02"],
+    "guid-03":
+        ["guid-05"],
+    "guid-05":
+        ["guid-06"],
+    "guid-07": ["guid-08-1", "guid-09-2", "guid-10-3", "guid-11-4"],
+    "guid-12": ["guid-13-1", "guid-14-2", "guid-15-3", "guid-16-4",
+                "guid-17-5", "guid-18-6", "guid-19-7", "guid-20-8",
+                "guid-21-9", "guid-22-10"],
+    "next": None
+}
+
+FAKE_BROKEN_LEGACY_DATA = {
+    "guid-23": [],
+    "guid-24": {},
+    "guid-25": None
+}
+
+FAKE_MIXED_LEGACY_DATA = {
+    "guid-26": [],
+    "guid-27": {},
+    "guid-28": None,
+    "guid-29": ["guid-30"]
+}
+
+
+@pytest.fixture
+def mock_amo_api_response(monkeypatch):
+    monkeypatch.setattr(requests, 'requests.get', lambda x, y: FAKE_LEGACY_DATA)
```

Comment:
On a second look, it seems that the second argument has an extraneous `requests.` prefixed since the module was already specified in the first argument.